### ISSUE TITLE
[Feat]: Stats form & stats metrics

### ISSUE THREAD

**Issue by widavies** [RobluScouting/RobluTodo#37]
The stat form is a new type of form that enables "statistical metrics". These metrics take "series" as inputs. In other words, a stat form works at a team level. Stats forms can take data points across all matches for that team and display them as a computed value or graphs. Also allow stats to work for a single match (or multiple) to compare teams. Like a mode where a user can select a list of teams to include and analyze. Another idea would be to set filters which define what is included in a graph or stat metric.

Some stats metrics:
- Bar graph
- Histogram
- Scatter plot
- Pie chart
- Avg, stddev, etc.
- min(), max(), minNonZeros(), and avgNonZeros()

**Comment by aidan-mundy** [comment]
Not sure that "forms" is the right name for this... Maybe "layouts"? Forms implies that they are filled out or submittable, whereas this is really just the layout of your statistics screen.

**Comment by widavies** [comment]
Yeah, good point. I like the name "views", [just like in SQL](https://en.wikipedia.org/wiki/View_(SQL)), you can create a view that isn't data by itself, but presents data in a certain way
